src/main/java/hoodini/Event.java

```diff
@@ -0,0 +1,48 @@
+package hoodini;
+
+/**
+ * Handles events that the user inputs
+ */
+public class Event extends Input {
+
+    private String from;
+    private String to;
+
+    /**
+     * Stores the String into Event Class
+     *
+     * @param input String input by user
+     */
+    public Event(String input) {
+        super(input.split(" ", 2)[1]
+                .split("/", 2)[0]);
+        this.from = input.split("/from", 2)[1]
+                .split("/to", 2)[0];
+        this.to = input.split("/to", 2)[1];
+    }
```

Comment:
I like that you use indentations for the multiple split calls as it makes it easier to read and understand 👍